### PR TITLE
feat: add known issue about v1.4.0 airgapped environment

### PR DESCRIPTION
**Pull request by FrankYang0529**
https://github.com/harvester/harvester/issues/7157

### CODE REVIEW

File: versioned_docs/version-v1.4/airgap.md [hunk 1]
```diff
@@ -49,12 +49,18 @@ When the nodes in the cluster do not use a proxy to communicate with each other,
...
\ No newline at end of file
+As a result, we recommend monitoring each [RKE2 release](https://github.com/rancher/rke2/releases) in your air gapped environment and pulling the required images into your private registry. Please refer to the **Harvester CCM & CSI Driver** with RKE2 Releases section on the [Harvester support matrix page](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-1-2/) for the best Harvester cloud provider and CSI driver capability support.
+
+## Known issues
+
+1. Missing rancher/rancher-agent:v2.9.2 image in the air-gapped environment.
+
+In harvester v1.4.0, the embedded rancher is v2.9.2. It introduced a new cronjob [rke2-machine-config-cleanup](https://github.com/rancher/rancher/commit/53ba791e11f98a5594c6fa8a30d321602483446c). It used rancher/rancher-agent:v2.9.2 as container image, but it's not included in the harvester v1.4.0 ISO. To avoid the issue, you can pull the image from the internet and load it into the air-gapped environment.
```
[the remark this answers is not in the document]
Comment (FrankYang0529) [reply versioned_docs/version-v1.4/airgap.md:68]: Fixed.

**Review by bk201 — APPROVED** [review]

File: versioned_docs/version-v1.4/airgap.md [hunk 2]
```diff
@@ -49,12 +49,34 @@ When the nodes in the cluster do not use a proxy to communicate with each other,
...
 
-As a result, we recommend monitoring each [RKE2 release](https://github.com/rancher/rke2/releases) in your air gapped environment and pulling the required images into your private registry. Please refer to the **Harvester CCM & CSI Driver** with RKE2 Releases section on the [Harvester support matrix page](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-1-2/) for the best Harvester cloud provider and CSI driver capability support.
\ No newline at end of file
+As a result, we recommend monitoring each [RKE2 release](https://github.com/rancher/rke2/releases) in your air gapped environment and pulling the required images into your private registry. Please refer to the **Harvester CCM & CSI Driver** with RKE2 Releases section on the [Harvester support matrix page](https://www.suse.com/suse-harvester/support-matrix/all-supported-versions/harvester-v1-1-2/) for the best Harvester cloud provider and CSI driver capability support.
+
+## Known issues
+
+---
+
+### 1. Missing rancher/rancher-agent:v2.9.2 image in the air-gapped environment. (Issue [#7157](https://github.com/harvester/harvester/issues/7157))
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.4/airgap.md:66]:
```suggestion
### 1. Missing rancher/rancher-agent:v2.9.2 image affects deployment in air-gapped environments. (Issue [#7157](https://github.com/harvester/harvester/issues/7157))
```

File: versioned_docs/version-v1.4/airgap.md [hunk 3]
```diff
@@ -49,12 +49,34 @@ When the nodes in the cluster do not use a proxy to communicate with each other,
...
+
+In harvester v1.4.0, the embedded rancher is v2.9.2. It introduced a new cronjob [rke2-machine-config-cleanup](https://github.com/rancher/rancher/commit/53ba791e11f98a5594c6fa8a30d321602483446c). It used rancher/rancher-agent:v2.9.2 as container image, but it's not included in the harvester v1.4.0 ISO. To avoid the issue, you can pull the image from the internet and load it into the air-gapped environment.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.4/airgap.md:68]:
```suggestion
Rancher v2.9.2, which is embedded in Harvester v1.4.0, introduced a new cronjob ([`rke2-machine-config-cleanup`](https://github.com/rancher/rancher/commit/53ba791e11f98a5594c6fa8a30d321602483446c)). This cronjob uses a container image (`rancher/rancher-agent:v2.9.2`) that is not included in the Harvester v1.4.0 ISO. To mitigate the issue, you can pull the image from the internet and then load it in the air-gapped environment.
```

File: versioned_docs/version-v1.4/airgap.md (hunk 2 above)
Comment (jillian-maroket) [reply versioned_docs/version-v1.4/airgap.md:66]: I'm not sure if deployment is blocked completely so I used "affects".

**Review by jillian-maroket — COMMENTED** [review]
Review done

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**